scripts/pylint_extensions.py

```diff
@@ -17,17 +17,33 @@
 """Implements additional custom Pylint checkers to be used as part of
 presubmit checks.
 """
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import division  # pylint: disable=import-only-modules
+from __future__ import print_function  # pylint: disable=import-only-modules
 
 import re
 
 import astroid
-import docstrings_checker  # pylint: disable=relative-import
-
 from pylint import checkers
 from pylint import interfaces
 from pylint.checkers import typecheck
 from pylint.checkers import utils as checker_utils
 
+import python_utils  # isort:skip
+from . import docstrings_checker  # isort:skip
+
+
+def read_from_node(node):
+    """Returns the data read from the ast node in unicode form.
+    Args:
+        node: astroid.scoped_nodes.Function. Node to access module content.
+    Returns:
+        unicode. The data read from the ast node.
```

Comment:
This is returning a list, not a unicode. Fix?

Reply:
Done